src/Lecture1.hs

```diff
@@ -77,7 +80,8 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+minmax :: Int  -> Int -> Int -> Int
+minmax x y z = maximum [x, y, z] - minimum [x, y, z]
```

Comment:
Your implementation is so short and elegant that you don't need local variables here 😅